### PR TITLE
Existing claim update

### PR DESCRIPTION
**Pull request by Shreyasks1729**
I'm not sure if it should be a new version of the chart but this should enable us to use the existing claims.

### CODE REVIEW

**Comment by femilofin** [comment]
@Shreyasks1729 You need to do the following:
- Bump up the version [here](https://github.com/eHealthAfrica/helm-charts/blob/master/src/generic3/Chart.yaml) to 0.3.2
- Run ./update.sh to generate chart for latest version
- Update src/generic3/VERSIONS.md to include the changes made on the chart

**Comment by Shreyasks1729** [comment]
done

File: src/generic3/templates/deployment.yaml [hunk 1]
```diff
@@ -285,7 +285,10 @@ spec:
         - mountPath: {{ .Values.app.volume.mountPath }}
           name: {{ template "generic.name" . }}-data
         {{ end }}
-
+        {{ if .Values.app.volume.existingClaim }}
+        - mountPath: {{ .Values.app.volume.mountPath }}
+          name: pvcname
```
Comment (wombolo) [inline src/generic3/templates/deployment.yaml:290]: Can this value `pvcname` be specified in the `values.yml`

File: src/generic3/templates/deployment.yaml [hunk 2]
```diff
@@ -319,6 +322,10 @@ spec:
         - mountPath: {{ .Values.app.volume.mountPath }}
           name: {{ template "generic.name" . }}-data
         {{ end }}
+        {{ if .Values.app.volume.existingClaim }}
+        - mountPath: {{ .Values.app.volume.mountPath }}
+          name: pvcname
```
Comment (wombolo) [inline src/generic3/templates/deployment.yaml:327]: Same concern here

**Review by wombolo — COMMENTED** [review]
I left some concerns with the pvc name specification

File: src/generic3/templates/deployment.yaml (hunk 1 above)
Comment (Shreyasks1729) [reply src/generic3/templates/deployment.yaml:290]: we can add it if needed but I don't think it's required only the claimName matters.

**Review by wombolo — APPROVED** [review]

**Pull request merged**